examples/reference/widgets/ChatEntry.ipynb

```diff
@@ -0,0 +1,218 @@
+{
```

Comment:
When I try this example

```python
import openai
import panel as pn

async def openai_callback(contents, user, instance):
    response = await openai.ChatCompletion.acreate(
        model="gpt-3.5-turbo",
        messages=[{"role": "user", "content": contents}],
        stream=True,
    )
    message = ""
    async for chunk in response:
        message += chunk["choices"][0]["delta"].get("content", "")
        yield message

chat_feed = pn.widgets.ChatFeed(callback=openai_callback, width=800).servable()
chat_feed.send("Have you heard of HoloViz Panel?")
```

I get a horizontal scrollbar. Never use horizontal scrollbars. Users don't like them 😄 

![image](https://github.com/holoviz/panel/assets/42288570/066091d9-0bf9-4533-878c-ce02959630df)

Reply:
It should also be possible to change the window width dynamically and the layout will adjust without introducing horizontal scrollbars like below.

https://github.com/holoviz/panel/assets/42288570/e5dd1d27-02d2-4fe7-a377-29542b8f4ec4

```python
import openai
import panel as pn

async def openai_callback(contents, user, instance):
    response = await openai.ChatCompletion.acreate(
        model="gpt-3.5-turbo",
        messages=[{"role": "user", "content": contents}],
        stream=True,
    )
    message = ""
    async for chunk in response:
        message += chunk["choices"][0]["delta"].get("content", "")
        yield message

chat_feed = pn.widgets.ChatFeed(callback=openai_callback, sizing_mode="stretch_width").servable()
chat_feed.send("Have you heard of HoloViz Panel?")
```

Reply:
Okay this should be fixed; thanks for QAing!

https://github.com/holoviz/panel/assets/15331990/b04b2e92-5c25-48fe-bb66-72dd4e046ce7